src/transformers/models/clvp/modeling_clvp.py

```diff
@@ -0,0 +1,1445 @@
+# coding=utf-8
+# Copyright 2023 The OpenAI Team Authors and The HuggingFace Team. All rights reserved.
```

Comment:
Think this copyright needs to be updated (Betker isn't from OpenAI as far as I'm aware)

Reply:
Actually he recently joined OpenAI (as this [post](https://nonint.com/2022/10/28/ive-joined-openai/) says.) but I agree that we should remove OpenAI as he says [here](https://github.com/neonbjb/tortoise-tts#notice).